admin/load.php

```diff
@@ -636,3 +730,35 @@ function perflab_plugin_admin_notices() {
 	}
 }
 add_action( 'admin_notices', 'perflab_plugin_admin_notices' );
+
+/**
+ * Helper function to check if a plugin is installed.
+ *
+ * @since n.e.x.t
+ *
+ * @param string $plugin_slug The plugin slug.
+ */
+function is_plugin_installed( $plugin_slug ) {
+	$plugins = get_plugins();
+	return isset( $plugins[ $plugin_slug ] );
+}
```

Comment:
Based on my suggestions, I believe this function will only be used in a single place. Since it's very simple, I would suggest to remove it and instead inline that functionality in the one place where it's needed. Every function is another piece of API that we'll have to support, so it's best to have as many as needed, but as few as possible.

Last but not least, we shouldn't have an unprefixed function like this in our codebase.